templates/base.html.twig

```diff
@@ -74,6 +74,7 @@
                     {% endset %}
                     <li><a href="https://phabricator.wikimedia.org/maniphest/task/edit/form/1/?title=%5BBUG%5D%20...&description={{ issue_template|escape('url') }}&projects=community-tech,svg_translate_tool&priority=triage">{{ msg('report-issue') }}</a></li>
                     <li><a href="https://meta.wikimedia.org/wiki/Special:MyLanguage/Community_Tech/SVG_translation">{{ msg('help') }}</a></li>
+                    <li><a href="https://translatewiki.net/wiki/Special:Translate/svgtranslate?group=svgtranslate&language=en&filter=%21translated&action=translate">{{ msg('translatewiki') }}</a></li>
```

Comment:
This URL should probably just be `https://translatewiki.net/wiki/Special:Translate/svgtranslate`, so that the user's own settings are used for the target language (i.e. setting `language=en` just results in a warning because it's the same as the base language).